Makefile

```
@@ -1,54 +1,64 @@
+publish: build
+	uv run twine upload dist/* --verbose $(ARGS)
+
+# Publish to Test PyPI (https://test.pypi.org/) for testing purposes.
+test-publish: build
+	uv run twine upload -r testpypi dist/* --verbose $(ARGS)
+
+# Run example script with the package published to Test PyPI
+# --index-strategy unsafe-best-match
+# We need to force it to look for other dependencies from the regular pypi server.
+# We save the output to a directory, and delete it afterwards if successful.
+test-testpypi-artifact:
```

checkmake (0.3.2)

[warning] Target body for "test-testpypi-artifact" exceeds allowed length of 5 lines (10).

(maxbodylength)